domain-model/src/main/java/uk/gov/hmcts/cmc/domain/constraints/PhoneNumberConstraintValidator.java

```diff
@@ -24,7 +24,7 @@ private boolean isWithInLengthLimit(String normalized) {
 
     private String normalize(String value) {
         return value
-            .replaceAll("\\(|\\)| |-|\\+", "")
+            .replaceAll("[() \\-+]", "")
```

Comment:
Can we just get a tester to cast an eye on this?

Reply:
Added a note on the JIRA ticket